soymsg/pomsg/pomsg_test.go

```diff
@@ -13,7 +13,7 @@ func TestPOBundle(t *testing.T) {
 		t.Error(err)
 		return
 	}
-	var bundle = pomsgs.Bundle("zz")
+	var bundle = pomsgs.Bundle("en")
```

Comment:
For my understanding, why were these test changes necessary?

Reply:
the new x/text/language package will throw an error ("tag is well-formed but unknown") when it tries to parse a locale that isn't defined by ISO_3166-1. Rather than re-write everything in the test to be compatible, it was much easier to just use a legitimate language tag.

Reply:
Oh, I think I see what's happening. This test was correctly identifying a behavior change. It used to return an empty bundle when the locale was not present (a zero bundle is usable), but now it returns nil. I think that should be corrected.

Reply:
Figured out that "zz" was failing because I had included locale parsing in Load originally. With it now only being present in Bundle, zz is a valid .po file. It just won't be able to do any sort of fallback checking on a fake locale.